specification/subscription/resource-manager/Microsoft.Subscription/stable/2021-10-01/subscriptions.json

```diff
@@ -0,0 +1,1130 @@
+        "x-ms-examples": {
+          "GetAlias": {
+            "$ref": "./examples/getAlias.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Subscription details.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionAliasResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/aliasNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      },
+      "delete": {
+        "description": "Delete Alias.",
+        "operationId": "Alias_Delete",
+        "x-ms-examples": {
+          "DeleteAlias": {
+            "$ref": "./examples/deleteAlias.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Deleted the alias"
+          },
+          "204": {
+            "description": "Alias does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/aliasNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Subscription/aliases": {
+      "get": {
+        "description": "List Alias Subscription.",
+        "operationId": "Alias_List",
+        "x-ms-examples": {
+          "ListAlias": {
+            "$ref": "./examples/listAlias.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Subscription details.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionAliasListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ]
+      }
+    },
+    "/providers/Microsoft.Subscription/{subscriptionId}/acceptOwnership": {
+      "post": {
+        "description": "Accept subscription ownership.",
+        "operationId": "Subscription_AcceptOwnership",
+        "x-ms-examples": {
+          "AcceptOwnership": {
+            "$ref": "./examples/acceptSubscriptionOwnership.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=11&length=3)

what is the reason for the update? according to our RPC, the recommended async post response is 202 Accepted - posts are actions.

Reply:
Swagger validation tool (linter) required to.

Reply:
Interesting, @changlong-liu do you know why the linter rule is different than ARM RPC? https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#call-action-post-asynchronously

Reply:
Nvm, after looking at your screenshot, the error because you updated your spec but did not update or added an example for this new API using the appropriate HTTP response.

Reply:
Hi @jianyexi , would you help on Jorge's question about whether the async post response should be 201 or 202?

Reply:
I think both 202/201 are accepted in lint for post LRO

Reply:
Thanks @jianyexi!
Hi @Ochirkhuyag , since the swagger should be aligned with service implementation, want to double confirm what's the service behavior about this? 201 or 202?

Reply:
The service is to accept ownership for the subscription. It is responding 202 in previous version. So, I updated it to 201 due to Swagger Validation tool (linter) requiring to change it to 201 when I validate it before the PR. But now I reverted back to 202. No changes on this. Even though Swagger Validation tool (linter) requires to change it to 201, it is okay on validation of the PR.